martypy/ClientMV2.py

```diff
@@ -407,6 +408,95 @@ def is_conn_ready(self) -> bool:
             return False
         return self._initComplete and (self.lastSubscribedMsgTime is not None)
 
+    def _valid_addon(self, add_on: str) -> bool:
+        disco = {"00000087","00000088","00000089"}
+        for attached_add_on in self.get_add_ons_status().values():
+            if attached_add_on['name'] == add_on:
+                if attached_add_on['whoAmITypeCode'] in disco:
+                    return True
+                else:
+                    raise MartyCommandException("The add on name passed in is not a valid disco add on. Please check the add on name in the scratch app -> configure -> add ons")
+        raise MartyCommandException("The add on name passed in is not a valid add on. Please check the add on name in the scratch app -> configure -> add ons")
+
+    def _valid_hex(self, color_hex: str) -> bool:
```

Comment:
```suggestion
    def _is_valid_color_hex(self, color_hex: str) -> bool:
```

I was also thinking about merging `_valid_hex` and `_hex_to_bytes` into something like `_parse_color_hex`. I have a feeling it might work out nicer that way but not sure 🤔